src/reference/faq.rst

```diff
@@ -83,3 +85,34 @@ For those who really want to use the ``arm64`` Miniconda installer, it is still
    conda config --env --set subdir osx-64
 
 This will ensure that all commands in the active Conda environment are run using ``osx-64`` emulation, making it possible to install all the software required for Nextstrain to run.
+
+.. _what-happened-to-the-native-runtime:
+
+What happened to the ``native`` runtime?
+----------------------------------------
+
+Up until Nextstrain CLI version 5.0.0, the name of the ``native`` runtime has been debated internally. In CLI version 5.0.0, **it was renamed to** ``ambient`` and we will use the new name going forwards.
+
+"Native" as a software term is typically used to describe software that can run without emulation, in other words optimized for your computer's processor.
+
+The ``ambient`` is "native" in that sense, but it puts all the software maintenance burden on the user. This means:
```

Comment:
```suggestion
The ``ambient`` runtime is "native" in that sense, but it puts all the software maintenance burden on the user. This means:
```

Reply:
Done in [force-push](https://github.com/nextstrain/docs.nextstrain.org/compare/6defc78df4d90e9e7e2a3d3bb2a8f4cd00cfe0fa..e731803c2442314f9ecc0a7f0d2b2569adb5daa0).